docs/provisioners/helm.md

```diff
@@ -73,7 +83,7 @@ will be updated to Unpacked, indicating that all its contents have been stored o
 ```console
 $ kubectl get bundle -l app=my-ahoy
 NAME                 TYPE   PHASE      AGE
-my-ahoy-5764594dc8   http   Unpacked   33s
+my-ahoy-5764594dc8   git    Unpacked   33s
```

Comment:
Should this actually be `http`? The BD above this is using `http` and not `git`.

Reply:
Yes, that's right.  I edited some bundles to use http.  I changed this by mistake, too.  Thanks!